index.bs

```diff
@@ -1318,6 +1350,9 @@ Elected Groups Vacated Seats</h5>
 		<li>
 			the participant resigns, or
 
+		<li>
+			the participant is [=recalled=], or
```

Comment:
These "or" list items should be rewritten, with a new lead-in —
`	An [=Advisory Board=] or [=TAG=] participant's seat is vacated when any of the following occurs:`
— and all of the `, or` should be removed from the `<li>` that follow. GitHub won't let me make a suggestion that reaches so far above and below the lines changed by this PR. If need be, I'll submit a PR for this, but I'm hoping this comment will be sufficient for you to adjust #888.